account_permanent_lock_move/wizard/permanent_lock_date_wizard.py

```diff
@@ -0,0 +1,35 @@
+# © 2016 Camptocamp SA (Matthieu Dietrich)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class PermanentLockDateWizard(models.TransientModel):
+    _name = 'permanent.lock.date.wizard'
+
+    lock_date = fields.Date(string="Lock Date")
+    company_id = fields.Many2one(comodel_name='res.company',
+                                 string='Company')
+
+    @api.multi
+    def confirm_date(self):
+        self.ensure_one()
+        company = self.company_id
+        if (company.permanent_lock_date and
+                self.lock_date < company.permanent_lock_date):
+            raise UserError(
+                _("You cannot set the permanent lock date in the past.")
```

Comment:
I do not understand this test. You may safely ignore the remark as I am not a functional guy, but self.lock_date is not tested against present date here, so the message error message mentioning "in the past" looks wrong (if at least company.permanent_lock_date can be in the future, which looks possible to me here). Maybe this deserves a bit more precise documentation or a method docstring.

Reply:
done